File: Solution/keywords.py
def getKeyWord(sentence):
    sentenceGroup = sentence.split(" ");
    combinations = [];
    value = '';
    for h in range(0,len(sentenceGroup)):
        slice = sentenceGroup[h:]
        value = ' '.join(slice)
        if (value not in combinations and len(value) > 1):
            combinations.append(value)
        for t in range(len(slice), 0, -1):
            value = ' '.join(sentenceGroup[h:t])
            if (value not in combinations and len(value) > 1):
                combinations.append(value)
    for w in sentenceGroup :
        if (w not in combinations and len(w) > 1):
            combinations.append(w)
    return combinations

# print(getKeyWord('clean white paper'))

File: Solution/Data and Text Manipulation/keywords.js
function getKeyWord(sentence) {
    let sentenceGroup = sentence.split(" ");
    let combinations = [];
    let value;
    for (let h=0; h<sentenceGroup.length; h++) {
        let slice = sentenceGroup.slice(h)
        value = slice.join(' ')
        if (!combinations.includes(value) && value.length > 1) {
            combinations.push(slice.join(' '))
        }
        for (let t=slice.length; t>0; t--) {
            value = sentenceGroup.slice(h,t).join(' ')
            if (!combinations.includes(value) && value.length > 1) {
                combinations.push(value)
            }
        }
    }
    sentenceGroup.forEach(function(w){
        if (!combinations.includes(w) && w.length > 1) {
            combinations.push(w)
        }
    })
    return combinations;
  }
  
//   console.log(getKeyWord("clean white paper"))